src/components/ContentNode/Aside.vue

```diff
@@ -74,10 +74,14 @@ $aside-kinds: deprecated, experiment, important, note, tip, warning;
 // on the fact that a "doc-topic" class is applied to an ancestor element of
 // the aside in question
 .doc-topic aside {
-  border-radius: $border-radius;
+  border-radius: var(--aside-border-radius, $border-radius);
+  border-style: var(--aside-border-style, solid);
+  border-width: var(--aside-border-width,
+    $aside-width-border
+    $aside-width-border
+    $aside-width-border
+    $aside-width-left-border);
```

Comment:
It could be changed from the theme file, but if it isn't, then the default will still look the same as it did before, where there is a distinct variable for the left border, allowing for these to be using different values.